dashboard/app/jobs/evaluate_rubric_job.rb

```diff
@@ -228,25 +247,75 @@ def self.log_metrics(response)
     )
   end
 
-  RETRIES_ON_TIMEOUT = 2
+  ATTEMPTS_ON_TIMEOUT = 2
 
   # Retry just once on a timeout. It is likely to timeout again.
-  retry_on Net::ReadTimeout, Timeout::Error, wait: 10.seconds, attempts: RETRIES_ON_TIMEOUT do
+  retry_on Net::ReadTimeout, Timeout::Error, wait: 10.seconds, attempts: ATTEMPTS_ON_TIMEOUT do |job, error|
+    log_metric(metric_name: :TimeoutError)
+    log_to_firehose(job: job, error: error, event_name: 'timeout-error')
+  end
```

Comment:
the `retry_on` block is only executed if/after all N attempts fail. the metric and event names are updated to reflect this.